src/Applications/GEOSdas_App/ldas_add2rc

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+MYNAME="ldas_add2rc"
```

Comment:
A different name for this script might be more informative.  Maybe `reconcile_ldas_adas_rc`?

Reply:
Maybe we move "rc" to after "reconcile",  a name ending with  "rc" could be misread as a rc file.

Reply:
maybe `reconcile_ldas_adas_config` then?